### ISSUE TITLE
No longer getting completions at all

### ISSUE THREAD

**Issue by dccmmccd1** [Andersbakken/rtags#341]
I just pulled the latest version, because I wanted to get the async from #310 but I'm not getting any completions at all.
Here's the values of the vars I have that I thought might be relevant:

company-backends: (company-rtags)
rtags-completions-enabled: t
company-rtags-use-async: nil  // I tried both nill and t, none help me
company-idle-delay: 0
rtags-completions-timer-interval: 0

I'm calling rtags-diagnostics in after-init-hook and setting company-backends like this:
```
(add-hook 'c++-mode-hook
          (lambda ()
            (set (make-local-variable 'company-backends) '(company-rtags))))
```

I don't believe I made any other changes related to rtags/company.

**Comment by Andersbakken** [comment]
Hi

Sorry about the long delay. Which gcc version are you using? If it's <= 4.8 std::regex doesn't work properly and we relied on that in the parsing of the completion command. We no longer do this however. Have you been able to make it work? If not, are you able to update to the latest rtags and try again?

Anders

**Comment by Andersbakken** [comment]
Please reopen if this still is a problem.

**Comment by dccmmccd1** [comment]
Hey, 

Took me a while to want to try to tackle this again. Yes, it's still not working. I'm not sure how is the gcc version relevant, since I used clang to compile rtags (I forget if I used 3.5 or 3.6). But even that I think isn't relevant because I think there is something wrong with the emacs side of it. I just can't seem to get any completions, async or no async. Using ".", "->" or "::", nothing seems to work. When I put company-clang back on the company-backends list, it starts working (using company-clang obviously). 
Also, just to be sure I updated both my company and rtags packages today (emacs side, still using the same rc executable compiled when head was:

commit d4c7f33ed902e26767bb3874db935556896d4a0c
Author: Anders Bakken <agbakken@gmail.com>
Date:   Mon Apr 20 12:52:05 2015 -0700

    Fix #issue 339. Make sure default-directory exists before calling rc
    functions.

Any suggestions what I should be looking at? Where the problem could be?

Thanks!
